index.js

```diff
@@ -149,9 +147,18 @@ function fastifyView (fastify, opts, next) {
     return viewExt ? `.${viewExt}` : (includeViewExtension ? `.${extension}` : filextension)
   }
 
+  function isPathExcludedMinification (currentPath, pathsToExclude) {
+    return pathsToExclude.includes(currentPath)
+  }
+
+  function useHtmlMinification (globalOpts, requestedPath) {
+    return globalOptions.useHtmlMinifier &&
+      (typeof globalOptions.useHtmlMinifier.minify === 'function') &&
+      !isPathExcludedMinification(requestedPath, globalOptions.pathsToExcludeHtmlMinifier || [])
+  }
   // Gets template as string (or precompiled for Handlebars)
   // from LRU cache or filesystem.
-  const getTemplate = function (file, callback) {
+  const getTemplate = (file, callback, requestedPath) => {
```

Comment:
Is the arrow function needed?

Reply:
Absolutely not, I replaced it with a normal function